site/content/en/docs/Installation/Terraform/gke.md

```diff
@@ -96,6 +96,14 @@ Configurable parameters:
 - gameserver_maxPort - the upper bound of the port range which gameservers will listen on (default is "8000")
 - gameserver_namespaces - a list of namespaces which will be used to run gameservers (default is `["default"]`). For example `["default", "xbox-gameservers", "mobile-gameservers"]`
 - force_update - whether or not to force the replacement/update of resource (default is true, false may be required to prevent immutability errors when updating the configuration)
+{{% feature publishVersion="1.28" %}}
+- location - the name of the [location](https://cloud.google.com/compute/docs/regions-zones) you want your cluster  to eb created in
```

Comment:
Better to callout the default values, like the other fields.

Reply:
Done!